libndt7/src/main/java/io/ooni/mk/nuvolari/libndt7/Client.java

```diff
@@ -151,24 +240,20 @@ private void periodic() {
 
     private String makeQuery() {
```

Comment:
These settings are gone now

Reply:
What should I keep vs remove?  Remove both `adaptive` and `duration`?

Reply:
Yes, both